Package.swift

```diff
@@ -31,6 +31,7 @@ let package = Package(
   ],
   dependencies: [
     .package(url: "https://github.com/apple/swift-package-manager.git", from: "0.1.0"),
+    .package(url: "https://github.com/Alex-Ozun/XcodeProj.git", .branch("master")),
```

Comment:
I've added my fork here, because I had to [make a small change](https://github.com/Alex-Ozun/XcodeProj/commit/594baa8063a0add8a2eee3a323e42fb5cb5df2e7) to `XcodeProj` to get me what I needed. If we decide to use `XcodeProj`, we will need to take this into consideration - either make another fork or submit PR (I plan to anyway).